knimin/templates/pm_plate_list.html

```diff
@@ -0,0 +1,173 @@
+<!-- The Plate Mapper module -->
+<!-- Plate list -->
+
+{% extends logged_in_index.html %}
+{% block head %}
+{% from json import dumps %}
+<script type="text/javascript" src="/static/js/labmin.js"></script>
+<script type="text/javascript" src="/static/vendor/js/jquery.validate.min.js"></script>
+<script type="text/javascript" src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<link rel="stylesheet" type="text/css" href="/static/vendor/css/jquery.dataTables.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
```

Comment:
@qiyunzhu you can create a folder in the `vendor` directory called font-awesome, where all the files needed for font-awesome are included. If in the future we don't want to use those icons anymore we can just remove that folder. Also, which is the LICENSE of font-awesome? We need to make sure that its LICENSE is compatible with our project and we may need to include a copy of their LICENSE in our repo, as it is done for other projects.